CONTRIBUTING.md

```diff
@@ -62,10 +74,15 @@ npm run test:watch
 This codebase adheres to the [Airbnb Styleguide](https://github.com/airbnb/javascript) and is
 enforced using [ESLint](http://eslint.org/).
 
+As with the test suite, linting rules apply to *built* Enzyme.
```

Comment:
This is maybe only partially correct. I believe the `import/*` eslint rules rely on built enzyme because they actually look at the filesystem and require that the built files be there, but linting doesn't actually run on the built files, they run on the src files... but they require the built files to be there. Kind of annoying that it works that way but hey, this is the babel-compiled world we live in...

Reply:
Good point - I've added some clarification to that.